docs/testrunner-toolkit.md

```diff
@@ -4,50 +4,30 @@ title: Using Testrunner Toolkit
 sidebar_label: Using Testrunner Toolkit
 ---
 
+<span className="sauceGold">BETA FEATURE</span><p/>
+
 import Tabs from '@theme/Tabs';
 import TabItem from '@theme/TabItem';
 
 
-Sauce Labs Testrunner Toolkit is designed to support native Javascript frameworks through a containerized testing solution that simplifies user set up and speeds up test execution time. 
-
-
-This approach combines the power and expressiveness of different test frameworks with the dashboards, infrastructure, and analytics of Sauce Labs.
-
-
-Most importantly, by running these tests through Sauce Labs, you can:
+Testrunner Toolkit is a containerized testing solution that allows you to get the benefits of the Sauce Labs platform while maintaining your test suites in your favorite JavaScript framework.  
```

Comment:
We are not JavaScript exlusive though. Especially with the addition of espresso (Java) and xcui (Swift).

Reply:
Yep. Will edit that to be less rigid.